frappe/email/doctype/email_account/email_account.json

```diff
@@ -1130,6 +1130,39 @@
    "translatable": 0,
    "unique": 0
   },
+  {
+   "allow_bulk_edit": 0,
+   "allow_in_quick_entry": 0,
+   "allow_on_submit": 0,
+   "bold": 0,
+   "collapsible": 0,
+   "columns": 0,
+   "default": "0",
+   "fieldname": "no_smtp_authentication",
+   "fieldtype": "Check",
+   "hidden": 0,
+   "ignore_user_permissions": 0,
+   "ignore_xss_filter": 0,
+   "in_filter": 0,
+   "in_global_search": 0,
+   "in_list_view": 0,
+   "in_standard_filter": 0,
+   "label": "Disable SMTP server authentication",
```

Comment:
@aleksas Where is the modified date for changed DT? 
This is causing issues for all users.

Reply:
Added it manually to json. What should be done to resolve it quickly?